package.json

```diff
@@ -3,7 +3,9 @@
   "name": "stlite",
   "workspaces": [
     "packages/*",
-    "streamlit/frontend"
+    "streamlit/frontend",
```

Comment:
NOTE: `streamlit/frontend` is still needed here because the `streamlit-proto` make rule calls `yarn pbjs` in this directory, and `yarn run` needs to be run in a workspace directory to find the command binary location.